db/migrate/20240209035322_update_ingredients_in_breads.rb

```diff
@@ -0,0 +1,13 @@
+class UpdateIngredientsInBreads < ActiveRecord::Migration[7.0]
+  def up
+    execute <<-SQL
+      ALTER TABLE breads ALTER COLUMN ingredients TYPE text[] USING ARRAY[ingredients];
```

Comment:
Still wrapping my brain around how these work, but stoked that we figured out how to get arrays in a schema! Pretty sick.